eip-5564.md

```diff
@@ -55,11 +55,13 @@ A recipient MUST be able to check if a stealth address belongs to them by callin
 /// @param stealthAddress The recipient's stealth address.
 /// @param ephemeralPubKey The ephemeral public key used to generate the stealth address.
 /// @param viewingKey The recipient's viewing private key.
+/// @param spendingPubKey The recipient's spending public key.
 /// @return True if funds sent to the stealth address belong to the recipient.
 function checkStealthAddress(
   address stealthAddress,
   bytes memory ephemeralPubKey,
-  bytes memory viewingKey
+  bytes memory viewingKey,
+  bytes memory spendingPubKey
```

Comment:
I added the spendingPubKey function which is required in the last step to derive the stealth address.